server/src/command_table_restart.go

```diff
@@ -152,16 +158,31 @@ func commandTableRestart(s *Session, d *CommandData) {
 		})
 	}
 
-	// Generate a random name for the new game
-	newTableName := getName()
+	newTableName := ""
+
+	// Generate a new name for the game based on which iteration of the room this is
+	// For example, a game named "logic only" will be "logic only (#2)" and then "logic only (#3)"
+	if t.InitialName != "" {
```

Comment:
i dont think it is necessary to add a new field here, it strikes me as cluttering the table oject.
rather, can't we infer the initial name by using regex in e.g. the exact same way that you increment the number?

Reply:
No because then the game would be "Shared Replay for game #N" 

and I don't think old table structures are kept

Reply:
oh, i see, that is unfortunate. maybe we should just get rid of table names altogether

Reply:
Hyphenated doesn't use table names but the majority of non-hyphenated does

Reply:
"using" a table name isn't very specific. they might make a custom table name with a silly, arbitrary string, but that isn't actually adding any value to anybody. on the contrary, with the name column removed from the main lobby screen, you have much less clutter

Reply:
still needed for e.g. don't accidentally spectate a competition game, unless there's a big refactor in how the lobby is drawn (hide these games?).
also i wouldn't recommend disabling table names until people who don't use chat commands know they can friend people

Reply:
agreed, maybe something to think about in the future